src/rules/get-fullcontact-profile.js

```diff
@@ -6,46 +6,53 @@
  *
  * This rule gets the user profile from FullContact using the e-mail (if available).
  *
- * If the information is immediately available (signaled by a `statusCode=200`), it adds a new property `fullContactInfo` to the user_metadata and returns. Any other conditions are ignored.
+ * If the information is immediately available (signaled by a `statusCode=200`), it adds a new property `fullcontact` to the user_metadata and returns. Any other conditions are ignored.
  *
- * See [FullContact docs](http://www.fullcontact.com/developer/docs/) for full details.
+ * See [FullContact docs](https://dashboard.fullcontact.com/api-ref) for full details.
  *
  */
 
 function (user, context, callback) {
+  // The following two configuration keys could be added to the Rules configuration object at https://manage.auth0.com/#/rules/
+  // See https://auth0.com/docs/rules/guides/configuration for details
+  
+  // Get FullContact API key: https://dashboard.fullcontact.com/
   const FULLCONTACT_KEY = configuration.FULLCONTACT_KEY;
+  
+  // Create a Slack app and get a hook URL: https://api.slack.com/messaging/webhooks#getting-started  
   const SLACK_HOOK = configuration.SLACK_HOOK_URL;
 
   const slack = require('slack-notify')(SLACK_HOOK);
 
   // skip if no email
   if (!user.email) return callback(null, user, context);
 
-  // skip if fullcontact metadata is already there
+  // skip if FullContact metadata is already there
   if (user.user_metadata && user.user_metadata.fullcontact) return callback(null, user, context);
 
-  request.get('https://api.fullcontact.com/v2/person.json', {
-    qs: {
-      email:  user.email,
-      apiKey: FULLCONTACT_KEY
+  request.get('https://api.fullcontact.com/v3/person.enrich', {
```

Comment:
@rosnovsky I had a second look at the FullContact V3 API and it appears to be a POST request: https://dashboard.fullcontact.com/api-ref#person-enrichment

```
curl -X POST https://api.fullcontact.com/v3/person.enrich \
-H "Authorization: Bearer {Your API Key}" \
-H "Content-Type: application/json" \
-d '{"email":"bart@fullcontact.com","twitter":"@bartlorang","fullName":"Bart Lorang"}'
```